src/components/footer.tsx

```diff
@@ -29,23 +38,46 @@ export const FooterLanguageToggle = () => {
           "button eyebrow is-small is-justify-content-center",
           locale === "es" && "is-selected"
         )}
+        onClick={() =>
+          logAmplitudeEvent("languageToggle", {
+            ...pageInfo,
+            switchToLanguage: "es",
+            location: "footer",
+          })
+        }
       >
         Español
       </LocaleToggle>
     </div>
   );
 };
 
-const FooterLink: React.FC<{ link: LinkWithLabel }> = ({ link }) =>
-  link[0].charAt(0) === "/" ? (
-    <Link className="jf-footer-page-link no-underline" to={link[0]}>
+const FooterLink: React.FC<{ link: LinkWithLabel }> = ({ link }) => {
+  const pageInfo = getPageInfo();
+  const logEvent = () =>
+    logAmplitudeEvent("pageLink", {
+      ...pageInfo,
+      page: link[1],
+      location: "footer",
+    });
+  return link[0].charAt(0) === "/" ? (
+    <Link
```

Comment:
should the Link component always log to amplitude?

Reply:
I had been thinking that could be left to google analytics, and with amplitude just focus on these more specific ones folks had questions about. Also since these same links appear in both the header and footer of each page, I thought with amplitude we could more easily distinguish by adding header/footer as a property.) 
But I think you're right that it might be better to just have them all in amplitude as well. I'll add that.

Reply:
oh, definitely agree that it'd be nice to distinguish the header clicks from the footer though... not the nicest, but could we pass an object with additional telemetry data through the link props? or if we gave each one a link id / key or something, that might also help differentiate?